src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/SemanticTokenTestBase.cs

```diff
@@ -67,6 +67,22 @@ internal void AssertSemanticTokensMatchesBaseline(IEnumerable<int>? actualSemant
             var semanticIntStr = semanticFile.ReadAllText();
             var semanticArray = ParseSemanticBaseline(semanticIntStr);
 
+            if (semanticArray is null && actual is null)
```

Comment:
This isn't strictly mandatory, but these arrays can get really long and the messages it leaves aren't readable, so I did a bit of extra work to check it in a way that's useful.